### ISSUE TITLE
Call to widget destroy function throws error: topSpacerClass is undefined

### ISSUE THREAD

**Issue by ronroberts** [watusi/jquery-mobile-iscrollview#116]
On Android, I have a Cordova-based application that is hooking all textarea and text input controls; click event. It calls a routine that calls the destroy method of two iscrollview controls resulting in this error (in the first such call). In terms of options, all were at their defaults except: 

```
    $.mobile.iscrollview.prototype.options.vscroll = true;
    $.mobile.iscrollview.prototype.options.hscroll = false;
    $.mobile.iscrollview.prototype.options.preventPageScroll = false;
    $.mobile.iscrollview.prototype.options.resizeWrapper = false;           
```

**Comment by ronroberts** [comment]
Solution to this is to change _undoAddSpacers to the following:

```
this.$wrapper.find(this.options.topSpacerClass).remove();
this.$wrapper.find(this.options.bottomSpacerClass).remove();
```

**Comment by jtara** [comment]
Fixed in 8d187b1
